ansible/Makefile

```diff
@@ -10,7 +10,7 @@ hosts:
 	echo -e "localhost ansible_connection=local\n\n[convergence_base]\nlocalhost" > $@
 
 local-defaults.yaml:
-	touch $@
+	echo -e "vars:" > $@
```

Comment:
do we want to use >> or perhaps even better add a conditional to avoid overwriting the file contents should there be any?

Reply:
This will not override the file content if one exists:

```
$ make local-defaults.yaml
echo -e "vars:" > local-defaults.yaml
$ make local-defaults.yaml
make: 'local-defaults.yaml' is up to date.
$ echo " my var: one" >> local-defaults.yaml 
$ make local-defaults.yaml
make: 'local-defaults.yaml' is up to date.
```

Reply:
this PR fixes things being broken out of the box, so i think we can go with it for now